### PR TITLE
fix(tools): update CODEOWNERS relative path of moved package

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Current Behavior: 
The `move-packages` nx generator was not updating the CODEOWNERS file with the moved package's new relative path

## New Behavior:
`move-packages` generator now updates CODEOWNERS file after package move.

## Fixes:
Follow-up to https://github.com/microsoft/fluentui/pull/22506#discussion_r857426868

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/78da3be7484d156379a105048c535767f7e1b85f'>78da3be7484d156379a105048c535767f7e1b85f</a></sub>
<div id="bundleSizeComment9423"/>

File: tools/generators/move-packages/index.spec.ts [hunk 1]
```diff
@@ -49,6 +49,9 @@ describe('move-packages generator', () => {
     };
     tree.write(`nx.json`, serializeJson(nxJsonConfig));
 
+    const codeOwners = `packages/test @dummyOwner`;
```
Comment (Hotell) [inline tools/generators/move-packages/index.spec.ts:52]: consider leveraging this https://github.com/microsoft/fluentui/blob/master/tools/utils-testing.ts#L5

File: tools/generators/move-packages/index.spec.ts [hunk 2]
```diff
@@ -130,6 +133,14 @@ describe('move-packages generator', () => {
       const storybookMainJsFile = tree.read(storybookMainJsPath, 'utf8') as string;
       expect(storybookMainJsFile.split(' ').filter(s => s.includes(newPath))).toHaveLength(2);
     });
+    it('should update the CODEOWNERS file with the new relative path', async () => {
+      await generator(tree, options);
+      const newPath = `packages/${options.destination}`;
+      const oldPath = `packages/test`;
+      const codeOwnersFile = tree.read(joinPathFragments('/.github', 'CODEOWNERS'), 'utf8') as string;
+      expect(codeOwnersFile.split(' ').filter(s => s === oldPath)).toHaveLength(0);
```
Comment (Hotell) [inline tools/generators/move-packages/index.spec.ts:141]: **suggestion:**

can we simplify this test/make it more obvious ? something like following:

```ts
const oldOwnerDeclaration = `packages/test  @dummyOwner`;
const newOwnerDeclaration = `packages/${options.destination}  @dummyOwner`;

let codeOwnersFile = tree.read(joinPathFragments('/.github', 'CODEOWNERS'), 'utf8');
expect(codeOwnersFile.indexOf(oldOwnerDeclaration) !== -1).toBeTrue();

await generator(tree, options);

codeOwnersFile = tree.read(joinPathFragments('/.github', 'CODEOWNERS'), 'utf8');
expect(codeOwnersFile.indexOf(oldOwnerDeclaration) !== -1).toBeFalse();
expect(codeOwnersFile.indexOf(newOwnerDeclaration) !== -1).toBeTrue();
```

Comment (TristanWatanabe) [reply tools/generators/move-packages/index.spec.ts]: Done in [e732115](https://github.com/microsoft/fluentui/pull/22624/commits/e732115e91537fdafe671341f4d475eeeb16db82)

**Review by Hotell — APPROVED** [review]
🙌

**Pull request merged**